browser/react/components/Login.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { login, logout } from '../../redux/reducers/auth';
+
+/* ----------------- COMPONENT ------------------ */
+
+class Login extends React.Component {
+  constructor (props) {
+    super(props);
+    this.onLoginSubmit = this.onLoginSubmit.bind(this);
+  }
+
+  // Set the background style & size for just this component
+  componentDidMount () {
+    document.body.style.background = 'url(/images/background.png) no-repeat top center fixed';
+    document.body.style.backgroundSize = 'cover';
+  }
+
```

Comment:
I've added a class name to the scenes equal to the href suffix. e.g. the <a-scene> returned by 'localhost:1337/beth/' has a class of 'beth'. Sounds like you won't need it, but it'll be there when my stuff gets merged in.